README.md

```diff
@@ -55,17 +56,19 @@ Context specific secrets for each environment that corresponds to a git branch (
 
 ## Self hosted runners
 
-We setup a bastian host in AWS for each environment which have access to the
-databases.  Then we associate those bastian hosts to
-[Github self-hosted runners](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/about-self-hosted-runners#about-self-hosted-runners).
-The update is executed from these runners.  Each runner corresponds
-to an environment which  corresponds to a git branch.  When a push happens on a
-branch (i.e. develop), the update is executed on the `agora-bastian-develop` runner
-which in turn updates the development database.
+[agora2-infra] repository deploys a bastian host in AWS for each environment which have access to
```

Comment:
Do we need the bastian for another reason? If not I think the runners should be on the private side.

Reply:
yes, that makes sense.  will make that change in agora2-infra

Reply:
https://github.com/Sage-Bionetworks/agora2-infra/pull/22